app/scripts/migrations/105.ts

```diff
@@ -1,21 +1,37 @@
 import { EthAccountType } from '@metamask/keyring-api';
-import { InternalAccount } from '@metamask/keyring-internal-api';
 import { sha256FromString } from 'ethereumjs-util';
 import { v4 as uuid } from 'uuid';
 import { cloneDeep } from 'lodash';
+import { Json } from '@metamask/utils';
 import { ETH_EOA_METHODS } from '../../../shared/constants/eth-methods';
 
 type VersionedData = {
   meta: { version: number };
   data: Record<string, unknown>;
 };
 
-type Identity = {
+export type Identity = {
   name: string;
   address: string;
   lastSelected?: number;
 };
 
+export type InternalAccountV1 = {
+  type: string;
+  id: string;
+  options: Record<string, Json>;
+  metadata: {
+    name: string;
+    importTime: number;
+    keyring: {
+      type: string;
+    };
+    lastSelected?: number;
+  };
+  address: string;
+  methods: string[];
+};
```

Comment:
suggestion - you can use `Omit` to just remove the `scopes` field from the type. That way also helps to identify the diff between both

```suggestion
export type InternalAccountV1 = Omit<InternalAccount, 'scopes'>
```

Reply:
That's what I wanted to do initially, unfortunately, this won't be truly "future-proof" since a new update to the `InternalAccount` will break this code again.

But... After saying that, updating it again later might be ok too... 🤔

If we add another field, we would just update the `Omit` again!

I might do that yes! I'll check if that builds fine (it should) and accept the suggestion afterward! Thanks. 😄

Reply:
Ah good point!! Then lets keep as that